### PR TITLE
Add sw ascii_converter and tb

### LINKED ISSUE #30: SW Ascii Converter

### PR DESCRIPTION
**Pull request by lrustand**
Closes #30

### CODE REVIEW

File: enigma/sw/ascii_converter.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    {
+        if (i < 26)
+            return i + 65;
+        return -1;
+    }
+
+    int from_ascii(char c)
+    {
+        if (c >= 'A' && c <= 'Z')
+            return c - 65;
```
Comment (Nelde95) [inline enigma/sw/ascii_converter.cpp]: \- 'A'

File: enigma/sw/ascii_converter.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        if (c >= 'a' && c <= 'z')
+            return c - 97;
```
Comment (Nelde95) [inline enigma/sw/ascii_converter.cpp]: \- 'a'

File: enigma/sw/ascii_converter.cpp [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#include "ascii_converter.h"
+
+namespace ascii_converter
+{
+    char to_ascii(int i)
+    {
+        if (i < 26)
```
Comment (Nelde95) [inline enigma/sw/ascii_converter.cpp:7]: if(i < LETTERS)

File: enigma/sw/ascii_converter_tb.cpp [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    }
+    else {
+        printf("Pass\n");
+    }
+
+    if (!test_conversion_consistency()) failed_testcases++;
+
+
+    if (failed_testcases) {
+        printf("converter_tb failed %d test(s).\n");
```
Comment (thomaskaafjeld) [inline enigma/sw/ascii_converter_tb.cpp]: Variabel i printf

**Pull request closed without merging**